### PR TITLE
[ETCM-396] faucet timeout

### PR DESCRIPTION
**Pull request by biandratti**
# Description

- Add a timeout setting in RpcClient used from the actor “FaucetHandler”.
- And change the property name man-backoff to max-backoff
- Change the message that we show when a failed connection between faucet and node. I added a pretty message.

# Proposed Solution

For default the rpc client used the idle-timeout, that represents 60 seconds. So, now we can set this configuration. 
When the actor FaucetHandler calls the node through the client rpc, now set a timeout of the request, represented by  “rpc-client.timeout”.
So, when the class “FaucetRpcService” calls the actor “FaucetHandler” set a timeout represented for the sum “response-timeout” (actor resolve) +  “rpc-client.timeout” (rpc client resolve).

When we have a timeout between the faucet and the node, in re response we receive the next message: "Faucet error: RPC request timeout"

# Testing

If you want to reproduce this case, you can set rpc-client.timeout with 1.milliseconds for example.

### CODE REVIEW

**Review by lemastero — APPROVED** [review]
LGTM 👍  This is a change in config. Probably worth to announce on proper channels.

**Review by ntallar — CHANGES_REQUESTED** [review]
Minor comment and LGTM! I'll create the corresponding PR on devops side and we'll discuss with them about re-deploying it

File: src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcService.scala [hunk 1]
```diff
@@ -16,7 +16,7 @@ class FaucetRpcService(config: FaucetConfig)(implicit system: ActorSystem)
     with FaucetHandlerSelector
     with Logger {
 
-  implicit lazy val actorTimeout: Timeout = Timeout(config.responseTimeout)
+  implicit lazy val actorTimeout: Timeout = Timeout(config.responseTimeout + config.rpcClient.timeout)
```
[the remark this answers is not in the document]
Comment (biandratti) [reply src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcService.scala:19]: Right, I can change the property too, with the same name “actor-communication-margin”.

[the remark this answers is not in the document]
Comment (biandratti) [reply src/main/scala/io/iohk/ethereum/faucet/jsonrpc/FaucetRpcService.scala]: Done

**Review by ntallar — APPROVED** [review]
LGTM!
```
 .\\            //.
. \ \          / /.
.\  ,\     /` /,.-
 -.   \  /'/ /  .
 ` -   `-'  \  -
   '.       /.\`
      -    .-
      :`//.'
      .`.'
      .' BP 
```

**Pull request merged**